mmv1/templates/terraform/custom_delete/kms_crypto_key_version.erb

```diff
@@ -0,0 +1,10 @@
+    cryptoKeyVersionId, err := parseKmsCryptoKeyVersionId(d.Id(), config)
```

Comment:
please standardize indentation on tabs or spaces for readability

Reply:
Nice catch! 
Actually this indentation problem is not visible in VS code, and make lint did not report any issue for this file. I will remember to double-check for readability for newly created file.